util/panics/panics.go

```diff
@@ -74,3 +75,12 @@ func exit(log *logs.Logger, reason string, currentThreadStackTrace []byte, gorou
 	os.Exit(1)
 	fmt.Print("After os.Exit(1)")
 }
+
+func handleSpawnedFunction(log *logs.Logger, stackTrace []byte, spawnedFunctionName string, spawnedFunction func()) {
+	goroutineID := atomic.AddUint64(&goroutineLastID, 1)
```

Comment:
According to this, goroutineID starts with 1 instead of 0. Consider working around that somehow.